src/screens/shared/components/FarmCard.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import '../../../styles/FarmCard.css';
+
+class FarmCard extends React.PureComponent {
+  render() {
+    const {
+      hubApproved,
+      gapApproved,
+      farmName,
+      farmerName,
+      farmLocation,
+      farmCover,
+      userIcon
+    } = this.props;
+
+    return (
+      <div className="farm-card">
```

Comment:
As for making a div clickable, can you add a `onClick={this.showModal()}`

Reply:
done